packages/node/README.md

```diff
@@ -277,7 +277,12 @@ module.exports = {
             }
           }]
         ]
-      }
+      },
+
+      // Example: add paths to `resolve.modules`
+      modules: [
+        neutrino.options.source,
```

Comment:
Could you update the `docs/` copy of the docs too? :-)

Reply:
Done!